README.md

```diff
@@ -45,3 +45,6 @@ python -m pytest tests/
 sphinx-apidoc -f -e -d 4 -o ./docs ./ladybug_charts
 sphinx-build -b html ./docs ./docs/_build/docs
 ```
+
+## Credits:
+This project is a derivative work of [Clima](https://clima.cbe.berkeley.edu/). We are grateful to the developers of the original work.
```

Comment:
We should mention the license for their project.

This project is a derivative work of [Clima](https://clima.cbe.berkeley.edu/) _which is available under an MIT license_.

Reply:
Complied here https://github.com/ladybug-tools/ladybug-charts/pull/16